web/themes/joinup/prototype/sass/base/_fontfaces.sass

```diff
@@ -0,0 +1,98 @@
+// Typography
```

Comment:
Please include all types of fonts.
As @brummbar suggested, you can use: https://google-webfonts-helper.herokuapp.com/fonts